Example/APMAlertController/ViewController.swift

```diff
@@ -194,25 +185,55 @@ class ViewController: UIViewController {
     func alertTextTitleAttributedMessage(_ indexPath: IndexPath) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/458898c990446c048fd2759812785a8e78fcda83/Example/APMAlertController/ViewController.swift#L185"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 44 lines</td>
    </tr>
  </tbody>
</table>